src/Input/Input.js

```diff
@@ -167,7 +167,8 @@ module.exports = kind(
 		onfocus: 'focused',
 		oninput: 'input',
 		onclear: 'clear',
-		ondragstart: 'dragstart'
+		ondragstart: 'dragstart',
+		onkeypress: 'testNumber'
```

Comment:
We don't want this running on every keypress. I'm skeptical about even fixing this as it is a browser-level issue with Safari. At most, we should only add this handler for Safari.